src/dom/ui-manip.ts

```diff
@@ -105,73 +106,49 @@ export function addLapDetails(runsJSON: IKartLapsPerRun, lapNum: number) {
   if (lapDetailsContainer) {
     lapDetailsContainer.innerHTML = `
   
-      <h3>Lap:  <span>${lapNum}</span></h3>
-      <h3>Max speed:  <span>${
+      <div class="mt-2 text-base">Lap:  <span class= "text-lg text-amber-300">${lapNum}</span></div>
+      <div class="mt-2 text-base">Max speed:  <span class= "text-lg text-amber-300">${
```

Comment:
Looking at the mt-2 you can wrap the section in a div and give the div a gap class.